src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityWrappers.cs

```diff
@@ -289,6 +289,7 @@ private static void InitializeAccessibilityDelegateTextInterface()
             ad.RemoveSelection   = AccessibilityRemoveSelectionWrapper;
             ad.SetCursorOffset   = AccessibilitySetCursorOffsetWrapper;
             ad.SetSelection      = AccessibilitySetSelectionWrapper;
+            ad.GetRangeExtents   = AccessibilityGetRangeExtentsWrapper;
```

Comment:
Can this go between `GetCursorOffset` and `GetSelection` so that we don't break the alphabetical order?

Reply:
Done. Thank you again.